site/content/docs/02-template-syntax.md

```diff
@@ -0,0 +1,592 @@
+---
+title: Template syntax
+---
+
+
+### Tags
+
+---
+
+A lowercase tag, like `<div>`, denotes a regular HTML element. A capitalised tag, such as `<Widget>`, indicates a *component*.
+
+```html
+<script>
+	import Widget from './Widget.svelte';
+</script>
+
+<div>
+	<Widget/>
+</div>
+```
+
+
+### Attributes
+
+---
+
+By default, attributes work exactly like their HTML counterparts.
+
+```html
+<div class="foo">
+	<button disabled>can't touch this</button>
+</div>
+```
+
+---
+
+As in HTML, values may be unquoted.
+
+```html
+<input type=checkbox>
+```
+
+---
+
+Attribute values can contain JavaScript expressions.
+
+```html
+<a href="page/{p}">page {p}</a>
+```
+
+---
+
+Or they can *be* JavaScript expressions.
+
+```html
+<button disabled={!clickable}>...</button>
+```
+
+---
+
+An expression might include characters that would cause syntax highlighting to fail in regular HTML, in which case quoting the value is permitted. The quotes do not affect how the value is parsed:
```

Comment:
Nit: You can still quote expressions even if HTML syntax highlighters wouldn't be confused. Maybe "in which case" -> "so"?

Reply:
👍